### DISCUSSION TITLE
Is python as frist language enough to get a job as SE?

### DISCUSSION THREAD

**Discussion by solaris5** [MomoSolaris8/superPower#5] [Q&A] [answered]
If someone not from the CS Major , e.g., from the Computer linguist or CS as an elective , is python enough to  get a job?

**Comment by MomoSolaris8** [comment, marked as answer]
Companies don't hire a single skill or just a language! They hire a set of skills!
Python is like plain Coffee, it doesn't  taste good! You need to add milk, sugar or caramel to make it tasty! Similarly, don't just learn python. Instead, you have to learn a little bit about other programming languages. You don't have to be master at  all of them- You need some like : 


1. TO build web develpoment using python, you need know the html, css and js! And the Framework like Django, Flask , etc.
2. Database(SQL) like MySQL, PostgreSQL, MongoDB etc.
